server/db/connectDB.js

```diff
@@ -1,6 +1,6 @@
 const mongoose = require('mongoose')
-
-const connectDB = (url)=>{
+const url=process.env.MONGO_URL;
```

Comment:
remove this line
no need to change connectDB.js file

Reply:
ohk any other change needed?